docker/docker-compose.test.yml

```diff
@@ -13,6 +13,17 @@ services:
     command: postgres -F
     network_mode: bridge
 
+  musicbrainz_db:
+    image: metabrainz/musicbrainz-test-database:beta
+    volumes:
+      - ../data/mbdata:/var/lib/postgresql/data/pgdata:z
+    environment:
+      PGDATA: /var/lib/postgresql/data/pgdata
+      MB_IMPORT_DUMPS: "true"
```

Comment:
This is not right, in a test environment, you would not want to download dumps and then import dumps. Make sure you know what the flag does. :)

Reply:
Yes, I was about to change this to false.